amundsen_application/static/js/components/common/ShimmeringTagListLoader/styles.scss

```diff
@@ -5,7 +5,7 @@
 
 $shimmer-loader-tag-height: 36px;
 $shimmer-loader-items: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14;
-$shimmer-loader-tag-max-width: 130;
+$shimmer-loader-tag-width: 90;
 $shimmer-loader-tag-min-width: 50;
```

Comment:
Same here

Reply:
`$shimmer-loader-tag-min-width` still used on line 14 for now, do you want to remove that too?